src/Microsoft.VisualStudio.SlowCheetah.VS/MultilingualResources/Microsoft.VisualStudio.SlowCheetah.VS.zh-Hans.xlf

```diff
@@ -238,7 +238,7 @@
         </trans-unit>
         <trans-unit id="PreviewToolNameLabel.Text" translate="yes" xml:space="preserve">
           <source>Preview Tool P&amp;ath: </source>
-          <target state="new">Preview Tool P&amp;ath: </target>
+          <target state="translated">预览工具路径：</target>
```

Comment:
Can there also be accelerators here? Similar to what to what was in other similar files: `(&amp;A)`. Also in other strings in this file